app/views/placements/support/providers/check.html.erb

```diff
@@ -12,88 +12,47 @@
         <%= t(".title") %>
       </label>
 
-      <dl class="govuk-summary-list">
-        <div class="govuk-summary-list__row">
-          <dt class="govuk-summary-list__key">
-            <%= t(".organisation_name") %>
-          </dt>
-          <dd class="govuk-summary-list__value">
-            <%= @provider_details.dig("attributes", "name") %>
-          </dd>
-        </div>
+      <%= govuk_summary_list do |summary_list| %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".organisation_name")) %>
+          <% row.with_value(text: @provider_details.dig("attributes", "name")) %>
+        <% end %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".ukprn")) %>
+          <% row.with_value(text: @provider_details.dig("attributes", "ukprn")) %>
+        <% end %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".accredited_provider_id")) %>
+          <% row.with_value(text: @provider_details.dig("attributes", "accredited_provider_id")) %>
+        <% end %>
+      <% end %>
+
+      <h2 class="govuk-heading-m govuk-!-margin-top-7">
+        <%= t(".contact_details") %>
+      </h2>
+
+      <%= govuk_summary_list do |summary_list| %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".email")) %>
+          <% row.with_value(text: @provider_details.dig("attributes", "email")) %>
+        <% end %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".telephone")) %>
+          <% row.with_value(text: @provider_details.dig("attributes", "telephone")) %>
+        <% end %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".website")) %>
+          <% row.with_value(text: @provider_details.dig("attributes", "website")) %>
```

Comment:
As with the school website, if you want the user to be able to click this: 
```suggestion
             <% row.with_value do %>
                <%= govuk_link_to(@provider_details.dig("attributes", "website"), @provider_details.dig("attributes", "website"), target: "_blank", rel: "noopener noreferrer") %>
            <% end %>
```

Reply:
resolved in [URL improvements](https://github.com/DFE-Digital/itt-mentor-services/pull/73/commits/bcf290bc40d8af1d1efd2ba3013b830fa3160207). Had to add a helper to format external links in the `link_to` if the string doesn't have `http://` or `https://`